client/vite.config.js

```diff
@@ -35,9 +35,10 @@ export default defineConfig({
 				data: {
 					VITE_CDN_URL: process.env.VITE_CDN_URL,
 					TACHI_NAME: process.env.TACHI_NAME ?? "Tachi",
-					VITE_ADDITIONAL_HEAD: process.env.VITE_GOATCOUNTER
+					THEME_INIT: '<script>var r=document.documentElement,l="light",d="dark",t=localStorage.theme||(window.matchMedia(`(prefers-color-scheme: ${d})`).matches?d:l);r.style.setProperty("color-scheme",t===l?l:d),r.setAttribute("data-bs-theme",t);</script>',
```

Comment:
don't minify the script at all -- you're just obfuscating what the code does

just write out the actual code here. you can use backticks `\`` to have newlines inside your strings.

Reply:
My approach was to impact the size of index.html as little as possible. If that's not a concern to you, then we can insert a less compact function into the browser if you want to maintain readability on our end.

Reply:
the difference isn't even a kilobyte, it doesn't matter.